src/index.js

```diff
@@ -2,35 +2,36 @@ const path = require('path');
 
 function install(moduleOptions) {
   const options = {
-    deferLoad: false,
+    preview: true,
+    components: true,
     ...moduleOptions,
+    ...(this.options.prismic || {}),
   };
 
   // Add in Prismic libraries to enable preview
-  if (typeof (this.options.head.__dangerouslyDisableSanitizersByTagID) === 'undefined') {
-    this.options.head.__dangerouslyDisableSanitizersByTagID = {};
-  }
+  if (options.preview) {
+    const repo = options.endpoint.replace(/^https?:\/\//, '').replace(/(\.cdn)?\.prismic.+/, '');
```

Comment:
Interesting, I'm wondering if might be easier if we just have the `repo` instead of the entire `endpoint`? Obviously a breaking change, but would make this a bit easier. The other positive is that we can control which version of the Prismic API this library uses.

Reply:
I thought about it too, but actually it's harder to get the `repo` from Prismic UI than the endpoint :/